tests/expense.test.js

```diff
@@ -38,7 +38,7 @@ describe('Expense', () => {
       expect(response.body)
         .toEqual(
           expect.objectContaining({
-            id: expect.any(Number),
+            id: expect.any(String),
```

Comment:
It seems like the ID type has been changed from Number to String. Make sure this change was intentional and is consistent across the codebase.

Reply:
It was intentional and in the app ids are strings.